cnf-testsuite/cluster_tools
# [1687] Manifest path fix for using as a shard and for project specs

> **Issues:** cncf/cnf-testsuite#1687

Fixes manifest path for using the project as a shard and for running the project specs.

### Validation: Running cnf-testsuite with the changes

**Validation steps:**
1. Remove `tools/cluster-tools` dir in the cnf-testsuite
2. Run `./cnf-testsuite routed_logs` test (uses ClusterTools underneath). Test should be skipped when fluendbit is installed
3. Install fluentd and run the routed_logs test again. Test should pass.
4. Uninstall fluentd and repeat step-2 again. Test should be skipped.

> *The branch with changes on cnf-testsuite has been renamed to bug/1687.*

<img width="993" alt="CleanShot 2022-10-14 at 15 51 13@2x" src="https://user-images.githubusercontent.com/84005/195824153-6d3fb4df-02e3-43a6-a2f3-d142f99083ed.png">


### Validation: Running project specs
<img width="954" alt="CleanShot 2022-10-14 at 15 49 55@2x" src="https://user-images.githubusercontent.com/84005/195823904-9757a404-868c-47bc-931c-092672bb1c46.png">
